src/apiBasePath.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * The api base path exported here is used to connect to the back-end API
+ * EC2 path: http://ec2-18-232-171-89.compute-1.amazonaws.com:8081/
+ * LocalHost path: http://localhost:8081/
+ */
+
+const apiBasePath = 'http://localhost:8081/';
```

Comment:
Leave out the ending '/' to be included if necessary in the string literal of requests.